hack/check-licenses.sh

```diff
@@ -27,6 +27,8 @@ while read line; do
           ;;
         github.com/letsencrypt/boulder)
           ;;
+        github.com/hashicorp/*)
+          ;;
```

Comment:
@thomasten should we just add needed packages here?

Reply:
I wanted to do it like this at first, but then suggested to Daniel to just add hashicorp/* for simplicity. But I don't have a strong opinion on this.